### ISSUE TITLE
Some issues in the tests

### ISSUE THREAD

**Issue by pgrt** [pcingola/SnpEff#310]
Hi,

I met some tiny issues while running the tests, please find them summarized in the enclosed patch. The first ones is related to files that -- I think -- are not in the source archive. The last one is related to numerical precision.

Best,
Pierre

[fix_tests.txt](https://github.com/pcingola/SnpEff/files/6229561/fix_tests.txt)

**Comment by pcingola** [comment]
Hi Pierre,
  Thank you for your analysis and patch.
  I'll make sure to include some of the needed test files in a tarball with precise
instructions on how to install the test datasets. Many integration tests rely on
large genome databases which I obviously don't want to include in a "standard"
distribution.
  I'll try to get back on this soon. Please let me know if there are deadlines / timelines
on your side and I'll do my best to accommodate.
  Thank you!

    Pablo

**Comment by pgrt** [comment]
Hi Pablo,

Many thanks for your reply! I understand very well the point with large databases.

On my side, there is no specific deadline right now. SnpEff 4.3t, which is the last tagged version in Github, will enter the development distributions of Debian in a few weeks. The first stable distribution of Debian it will enter will be released in 2023. So obviously we have some time, although some other software we would like to package depend on SnpEff and we will thus be happy to have it sooner than 2023.
In short: sticking to your own schedule will be fine for us!

Thanks a lot,
Pierre

**Comment by pcingola** [comment]
Great.
Yes, I expect to get this done before 2023 :-)
Thank you!

**Comment by pcingola** [comment]
Oh! I now understand why the confusion about 5.0 vs 4.3
Version 5.0 has been released for a while, but it looks like I haven't added the release tag in Github.
I'll release an incremental (5.0f) soon, so I'll make sure I update the tags this time.
Apologies for creating this confusion.

**Comment by pcingola** [comment]
Added documentation on testing here: https://pcingola.github.io/SnpEff/tests/

And a tarball with all the datasets here (link also in the previous doc):
https://snpeff.blob.core.windows.net/databases/data_test_5.0.tar

**Comment by pgrt** [comment]
Hi,

Thanks for the explanations about the upcoming 5.0f version, it is fine to me :)
Thanks also for the new documentation on testing, this is very helpful.

Best wishes,
Pierre
